nixos/modules/services/web-apps/audiobookshelf.nix

```diff
@@ -0,0 +1,97 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.audiobookshelf;
+in
+{
+  options = {
+    services.audiobookshelf = {
+      enable = mkEnableOption "Audiobookshelf, self-hosted audiobook and podcast server.";
+
+      package = mkOption {
+        description = "Audiobookshelf package to use";
+        default = pkgs.audiobookshelf;
+        defaultText = literalExpression "pkgs.audiobookshelf";
+        example = literalExpression "pkgs.audiobookshelf";
+        type = types.package;
+      };
+
+      dataDir = mkOption {
+        description = "Path to Audiobookshelf config and metadata.";
+        default = "/var/lib/audiobookshelf";
+        type = types.path;
+      };
```

Comment:
Is there a reason to have it outside of `/var/lib`?

Reply:
There is not. I have changed `dataDir` to be a name inside of `/var/lib`. This option is one of those nixpkgs conventions that lack standardization, so I find it difficult to know what to do with it.